pages/pipelines/functions.rst

```diff
@@ -322,6 +334,11 @@ contains
 
 Checks if ``value`` contains ``search``, optionally ignoring the case of the search pattern.
 
+Example::
+
+        // Check if the `computer.corp.net` is in the `hostname` field. Ignore case.
+        contains(to_string($message.hostname), "computer.corp.net", true)
```

Comment:
could you please change that to `example.org` as we use that all over the documentation.

Reply:
Done!